web/src/main/react-dashboard/src/components/ProductStatusView.js

```diff
@@ -70,17 +75,39 @@ class ProductStatusView extends Component {
     this.props.history.push(route);
   }
 
+    downloadReport(productName) {
+        let url = TESTGRID_CONTEXT + '/api/products/reports?product-name=' + productName;
+        fetch(url, {
+            method: "GET",
+            credentials: 'same-origin',
+            headers: {
+                'Accept': 'application/html'
+            }
+        }).then(response => {
+                if (response.status === HTTP_NOT_FOUND) {
+                    let errorMessage = "Unable to locate report in the remote storage, please contact the administrator";
+                    this.toggle(errorMessage);
+                } else  if (response.status !== HTTP_OK){
+                    let errorMessage = "Internal server error. Couldn't download the report at the moment, please " +
+                        "contact the administrator";
```

Comment:
End the error message with a dot since it is viewable to UI users?

Reply:
+1 fixed with 751dd40f5a5e69ca20eef7775268e501116da992